frontend/components/icons/ExternalLink.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+interface IExternalLinkProps {
+  size: "small" | "medium";
+}
+
+const SIZE_MAP = {
+  small: "12",
+  medium: "16",
+};
+
+const ExternalLink = ({ size = "small" }: IExternalLinkProps) => {
+  return (
+    <svg
+      width={SIZE_MAP[size]}
+      height={SIZE_MAP[size]}
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 16 16"
```

Comment:
TIL You need the original viewBox (not on svg export) or else it will do funky crops with width and height modifications